params/config.go

```diff
@@ -49,13 +38,13 @@ func isForked(fork_start, block_num types.BlockNum) bool {
 	return fork_start <= block_num
 }
 
-func (c *ChainConfig) Rules(num types.BlockNum) vm.Rules {
-	return vm.Rules{
-		IsHomestead:      isForked(c.HomesteadBlock, num),
-		IsEIP150:         isForked(c.EIP150Block, num),
-		IsEIP158:         isForked(c.EIP158Block, num),
-		IsByzantium:      isForked(c.ByzantiumBlock, num),
-		IsConstantinople: isForked(c.ConstantinopleBlock, num),
-		IsPetersburg:     isForked(c.PetersburgBlock, num) || c.PetersburgBlock == types.BlockNumberNIL && isForked(c.ConstantinopleBlock, num),
-	}
-}
+// func (c *ChainConfig) Rules(num types.BlockNum) vm.Rules {
+// 	return vm.Rules{
```

Comment:
Isn't it better to have some basic Californicum rule here and uncomment all related code?